### ISSUE TITLE
Provide more intro content in the docs

### ISSUE THREAD

**Issue by critesjosh** [AztecProtocol/dev-rel#139]
The following questions came up as Nico was reading our docs for onboarding
- what is a circuit?
- what is a contract?
- what does it mean to write a contract for aztec?

---

Additional categories to make intro content about:
- Portals

**Comment by signorecello** [comment]
I think this fits into the dev journey talk. I'd love to answer questions like `what is a contract` but I'm not sure they fit under any of our targeted personas for the docs.

Seems like this could be covered by @lisaaztec down the line in that new section on the website?

**Comment by signorecello** [comment]
But `what does it mean to write a contract for aztec` definitely could help a lot in the intro, specially when clearly defining what is client-side execution all about

**Comment by critesjosh** [comment]
Yes, I mean "What is a contract in the context of Aztec?" Our target audience right now is blockchain developers, and they have preconceived notions about what smart contracts are and how they work. These ideas are going to be quite a bit different for Aztec, where contracts are a collection of verification keys, references to note_hashes and nullifiers, as well as the familiar public state trees.
